app/src/main/kotlin/com/wire/android/ui/home/conversations/details/participants/GroupConversationParticipants.kt

```diff
@@ -100,12 +100,12 @@ fun MLSProgressIndicator(
         contentAlignment = Alignment.Center
     ) {
         LinearProgressIndicator(
-            progress = progress,
+            progress = { progress },
```

Comment:
I wonder if passing this as lambda will cause unnecessary recomposition?

Reply:
Actually it's the opposite, using the lambda will prevent recomposition. If progress value changes then only the lambda function will re-run not all the screen

Reply:
not really, a new state value a composition for the whole screen and when evaluating lambda objects it will always not be equal in this case (even if the value of progress did not change)

Reply:
The read of the value will be done in the child composable, so it won't effect as explained here
https://youtu.be/ahXLwg2JYpc?t=356

And the current one is deprecated: "Use the overload that takes progress as a lambda "